### PR TITLE
Scaffold page

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/tempus-client_v3/src/components/PageNavigation/PageNavigation.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
   position: relative;
   padding: 10px 20px;
   border: none;
+  margin-bottom: -1px;
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/components/PageNavigation/PageNavigation.scss:5]: I think it is better to set `box-sizing: border-box` for tabs and change `.tc__page-navigation__background::before`'s `top` and `bottom` to 0. Plus, those SVGs must be cut better to have a seamless shape match.

File: packages/tempus-client_v3/src/components/App/App.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  max-width: 1360px;
+  width: 100%;
+}
+
+.tc__app__body {
+  flex: 1 1 100%;
+  padding: 30px;
+  box-sizing: border-box;
+  width: 100%;
+  max-width: 1360px;
```
Comment (lazarmitic) [inline packages/tempus-client_v3/src/components/App/App.scss:45]: It seems `1360px` is used in few places, can we create a variable for it.

File: packages/tempus-client_v3/src/components/PageNavigation/PageNavigation.scss (hunk 1 above)
Comment (danieledamiani) [reply packages/tempus-client_v3/src/components/PageNavigation/PageNavigation.scss]: Checked with @Olafkuusik and some css were fixed, the SVG should be fine now

File: packages/tempus-client_v3/src/components/App/App.scss (hunk 2 above)
Comment (danieledamiani) [reply packages/tempus-client_v3/src/components/App/App.scss]: refactored

**Review by lazarmitic — APPROVED** [review]

**Pull request merged**